paddle/scripts/docker/build.sh

```diff
@@ -130,7 +130,7 @@ fi
 
 # generate deb package for current build
 # FIXME(typhoonzero): should we remove paddle/scripts/deb ?
-if [[ ${WITH_DEB:-OFF} == "ON" ]]; then
+if [[ ${WITH_DEB:-ON} == "ON" ]]; then
```

Comment:
Got that @helinwang enabled this flag in TeamCity, so I am good with this change.